app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/ChooseLanguageFinalPage.kt

```diff
@@ -0,0 +1,84 @@
+package com.example.awesomelanguagelearning.chooseLanguage.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.awesomelanguagelearning.R
+import com.example.awesomelanguagelearning.chooseLanguage.domain.entity.ChooseLanguagePageState
+import com.example.awesomelanguagelearning.core.ui.theme.AppTheme
+import com.example.awesomelanguagelearning.core.ui.views.HorizontalSpacer
+import com.example.awesomelanguagelearning.core.ui.views.TextButton
+import com.example.awesomelanguagelearning.core.ui.views.TextTitle
+
+@Composable
+fun ChooseLanguageFinalPage(
+    state: ChooseLanguagePageState,
+    onEvent: (ChooseLanguageEvent) -> Unit = {}
```

Comment:
@igel911 might want to pass modifier as a parameter here 👀

Reply:
@slothdeboss So you're a big fan of modifiers, I see.